utils.pwsh/Setup-Target.ps1

```diff
@@ -39,8 +39,8 @@ function Setup-BuildParameters {
     $VisualStudioData = Find-VisualStudio
 
     $VisualStudioId = "Visual Studio {0} {1}" -f @(
-        ([System.Version] $VisualStudioData.Version).Major
-        ([regex]::Match($VisualStudioData.Name, "\b\d{4}\b"))
+        $VisualStudioData.InstallationVersion.Major
+        ( $VisualStudioData.DisplayName -split ' ')[-1]
```

Comment:
There should either be a space at the end of this parenthetical, or we should remove the space at the beginning. We're not 100% consistent with this throughout, but we don't need to do a consistency sweep in this PR.

Reply:
I think we use spaces in conditionals, but don't use them when enclosing expressions.